packages/driver/cypress/e2e/e2e/security.cy.js

```diff
@@ -3,4 +3,21 @@ describe('security', () => {
     cy.visit('/fixtures/security.html')
     cy.get('div').should('not.exist')
   })
+
```

Comment:
Was the intentionally merged here?

I think something went wrong during the rebase, it looks like some extra files in `proxy` and this one got added.

Reply:
something is off as this branch is missing 478407ec38b305cb5a55add1f089fbaf1920ceec. What I might do to make this change set simpler is to create a new branch with a single commit of the changes based off `develop`. @zachz-indeed you would likely need to recreate the branch locally when this happens

Reply:
I think we need someone to own this, I think we should just make a new branch and pick the files we need. I can do it if no-one else has bandwidth, but probably not until the first week of Feb.

Reply:
I took care of it. [0843b0a](https://github.com/cypress-io/cypress/pull/23915/commits/0843b0a9198bfb8dd6ddde741b3641cb20d5e34c) is the commit with files we need with snapshots updated with the CI run in [f6aee7c](https://github.com/cypress-io/cypress/pull/23915/commits/f6aee7cc33b02b0a1184441b76f560f2370458f3). This should be ready for review, which I don't feel super comfortable reviewing simply because of how many changes I committed.

Reply:
I took a look, I think it looks fine. Any particular change you are concerned with? It's a lot of files, but all the changes are the same and what I'd expect.

Reply:
nothing in particular. I think the two src file updates make sense and the snapshots are G2G and match what is expected